code/game/objects/items/lighter.dm

```diff
@@ -52,6 +52,13 @@
 	)
 	update_appearance()
 
+/obj/item/lighter/examine(mob/user)
+	. = ..()
+	if(get_fuel() <= 0)
+		. += span_notice("It is out of lighter fluid! Refill it with welder fuel.")
```

Comment:
```suggestion
		. += span_warning("It is out of lighter fluid! Refill it with welder fuel.")
```

Reply:
I was torn on doing this just for the consistency of the tip staying blue, but if it's preferred I'll commit it yeah